optimum/exporters/openvino/utils.py

```diff
@@ -311,3 +312,16 @@ def save_preprocessors(
                 logger.error(f"Saving {type(processor)} failed with {ex}")
     else:
         maybe_save_preprocessors(model_name_or_path, output, trust_remote_code=trust_remote_code)
+
+
+SKIP_CHECK_TRACE_MODELS = ("deepseek", "deepseek-v2", "deepseek-v3")
+
+
+def allow_skip_tracing_check(library_name, model):
```

Comment:
only the model type looks needed here so I'd instead replace it with : 

```suggestion
def allow_skip_tracing_check(library_name: str, model_type: str):
```